dump/main.py

```diff
@@ -0,0 +1,233 @@
+            all_dbs.add(db_name)
+            
+    db_dict = {}  
+    for db_name in all_dbs:
+        if db_name is "CONFIG_FILE":
+            db_dict[db_name] = JsonSource()
+            db_dict[db_name].connect(plugins.dump_modules[module].CONFIG_FILE, namespace)
+        else:
+            db_dict[db_name] = RedisSource()
+            db_dict[db_name].connect(db_name, namespace)
+            
+    final_info = {}
+    for id in info.keys():
+        final_info[id] = {}
+        for db_name in info[id].keys():
+            final_info[id][db_name] = {}
+            final_info[id][db_name]["keys"] = []
+            final_info[id][db_name]["tables_not_found"] = info[id][db_name]["tables_not_found"]
+            for key in info[id][db_name]["keys"]:
+                final_info[id][db_name]["keys"].append({key : db_dict[db_name].get(db_name, key)})
+                 
+    return final_info
+
+def get_dict_str(key_obj):
+    table = []
+    for pair in key_obj.items():
+        table.append(list(pair))
+    return tabulate(table, headers=["field", "value"], tablefmt="psql")
+
+def get_keys(dump):
+    keys = []
+    for key_ in dump:
+        if isinstance(key_, dict) and key_:
+            keys.append(list(key_.keys())[0])
+        else:
+            keys.append(key_)
+    return keys
+
+def get_rid(redis_key, vidtorid):
+    matches = re.findall(r"oid:0x\w{1,14}", redis_key)
+    if matches:
+       vid = matches[0]
+       if vid in vidtorid:
+           return vidtorid[vid]   
+    return "Not Found"
+ 
+# print dump
+def print_dump(collected_info, table, module, identifier, key_map):
+    if not table:
+        click.echo(json.dumps(collected_info, indent=4))
+        return
+
+    top_header = [plugins.dump_modules[module].ARG_NAME, "DB_NAME", "DUMP"]
+    final_collection = []
+    for ids in collected_info.keys():
+        for db in collected_info[ids].keys():
+            total_info = ""
+
+            if collected_info[ids][db]["tables_not_found"]:
+                total_info += tabulate([collected_info[ids][db]["tables_not_found"]], ["Tables Not Found"], tablefmt="grid")
+                total_info += "\n"
+    
+            if not key_map:
+                values = []
+                hdrs = ["Keys", "field-value pairs"]
+                for key_obj in collected_info[ids][db]["keys"]:
+                    if isinstance(key_obj, dict) and key_obj:
+                        key = list(key_obj.keys())[0]
+                        values.append([key, get_dict_str(key_obj[key])])
+                total_info += str(tabulate(values, hdrs, tablefmt="grid"))
+            else:
+                temp = []
+                for key_ in collected_info[ids][db]["keys"]:
+                    temp.append([key_])
+                total_info += str(tabulate(temp, headers=["Keys Collected"], tablefmt="grid"))
+
+            total_info += "\n"
+            if "vidtorid" in collected_info[ids][db]:
+                table = []
```

Comment:
The name for this variable is same as one of the arguments to the API. Please change.